sass/main.scss

```diff
@@ -58,10 +58,21 @@
 
 
 .filter-group {
+  margin-top: 30px;
```

Comment:
Coo, just making sure, the sass versus scss is only if things don't break by doing so.